lib/tst_res.c

```diff
@@ -81,8 +81,13 @@ int TEST_ERRNO;
 	assert(strlen(buf) > 0);		\
 } while (0)
 
-#if defined(__ANDROID__) && !defined(PTHREAD_RECURSIVE_MUTEX_INITIALIZER_NP)
+#ifndef PTHREAD_RECURSIVE_MUTEX_INITIALIZER_NP
+#ifdef __ANDROID__
```

Comment:
indent the blocks so it's easier to follow

```
#ifndef ...
# ifdef ...
# else
# endif
#endif
```

Reply:
@vapier: done in new force pushed branch; sorry I know it is confusing but wasn't sure how the workflow for github is for LTP and had been requested it this way before in other projects (even if it somehow breaks github reviews)